backend/corpora/api_server/app.py

```diff
@@ -19,8 +19,14 @@ def create_flask_app():
     connexion_app = connexion.FlaskApp(f"{APP_NAME}-{DEPLOYMENT_STAGE}", specification_dir="backend/config")
     # From https://github.com/zalando/connexion/issues/346
     connexion_app.app.url_map.strict_slashes = False
-    swagger_spec_path = f"{APP_NAME}.yml"
-    connexion_app.add_api(swagger_spec_path, validate_responses=True)
+    dataportal_api = super(connexion.FlaskApp, connexion_app).add_api(
```

Comment:
why is this `super` call necessary?

Reply:
what about [this comment](https://github.com/spec-first/connexion/issues/512#issuecomment-570658527) on that issue's thread?

Reply:
It also takes care of the blueprint registration below, it seems. Correct me if I'm reading it wrong.

Reply:
Oh, but we have to have be able to specify the `url_prefix` when we make the call to `register_blueprint`. Nvm then.